sources/packages/backend/apps/api/src/route-controllers/announcements/announcements.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Get } from "@nestjs/common";
+import BaseController from "../BaseController";
+import { ApiTags } from "@nestjs/swagger";
+import { Announcements } from "@sims/sims-db";
+import { AnnouncementsService } from "../../services";
+import { Public } from "../../auth/decorators/public.decorator";
+
+@Controller("announcements")
+@ApiTags("announcements")
+export class AnnouncementsController extends BaseController {
+  constructor(private readonly announcementsService: AnnouncementsService) {
+    super();
+  }
+
+  @Public()
```

Comment:
I believe security-wise it must follow the same level as dynamic form controller. 

i.e. `@AllowAuthorizedParty` must be used, not the public.

![image](https://github.com/user-attachments/assets/933089cd-eefe-4425-a0c0-861e10ee9bac)

Reply:
Split into student and institution controllers